doc/guides/building-node-with-ninja.md

```diff
@@ -4,13 +4,16 @@ The purpose of this guide is to show how to build Node.js using [Ninja][], as
 doing so can be significantly quicker than using `make`. Please see
 [Ninja's site][Ninja] for installation instructions (Unix only).
 
-To build Node.js with ninja, there are 3 steps that must be taken:
+[Ninja][] is already supported at the Makefile, the only thing you need to
+execute is `./configure --ninja`, and this will configure the project to run
+the regular `make` commands with [Ninja][].
 
-1. Configure the project's OS-based build rules via `./configure --ninja`.
-2. Run `ninja -C out/Release` to produce a compiled release binary.
-3. Lastly, make symlink to `./node` using `ln -fs out/Release/node node`.
+For example, running `make` will execute `ninja -C out/Release` internally
+to produce a compiled release binary, and will also execute
+`ln -fs out/Release/node node`, so that you can execute `./node` at
+the project's root.
```

Comment:
```suggestion
For example, `make` will execute `ninja -C out/Release` internally
to produce a compiled release binary. It will also execute
`ln -fs out/Release/node node` so that you can execute `./node` at
the project's root.
```